src/rai_bench/rai_bench/tool_calling_agent/tasks/spatial.py

```diff
@@ -118,9 +136,43 @@ def __init__(
     def available_tools(self) -> List[BaseTool]:
         return [ReturnBoolResponseTool()]
 
-    def get_prompt(self):
+    @property
+    def optional_tool_calls_number(self) -> int:
+        return 0
+
+    def get_base_prompt(self) -> str:
         return self.question
 
+    def get_prompt(self):
+        if self.prompt_detail == "brief":
+            return self.get_base_prompt()
+        elif self.prompt_detail == "moderate":
+            return f"{self.get_base_prompt()} using visual analysis"
+        else:
+            return (
+                f"{self.get_base_prompt()} using the visual analysis system. "
+                "You can examine the provided image(s) carefully to identify relevant features, "
+                "analyze the visual content, and provide a boolean response based on your observations."
+            )
```

Comment:
I am wondering if these prompt shouldn't be placed somewhere near the "brief" prompt. Potentially the can be adjusted according to the spatial reasoning task. For example they can contain the information about what is regarded as left/right in the image or an example of a round shape

Reply:
As discussed, for tool calling bench, prompts with examples might not make too much sense.